src/uct/api/v2/uct_v2.h

```diff
@@ -109,6 +116,20 @@ typedef struct {
      */
     ucs_memory_type_t   remote_memory_type;
 
+    /**
+     * System device where the local memory type resides.
+     * Can be UCS_SYS_DEVICE_ID_UNKNOWN.
+     * This field must be initialized by the caller.
+     */
+    ucs_sys_device_t    local_sys_device;
```

Comment:
how do I query ucs_sys_device_t to fill in this parameter ?

Reply:
@shamisp the systemd device id can come from several places:
1. `uct_tl_resource_desc_t::sys_device` https://github.com/openucx/ucx/blob/b75809faa3e8c2015c5f5ab529ae90b0402c7ff0/src/uct/api/uct.h#L327
2. `uct_md_mem_query()` and https://github.com/openucx/ucx/blob/b75809faa3e8c2015c5f5ab529ae90b0402c7ff0/src/uct/api/uct.h#L1437
3. search for device directly by BDF https://github.com/openucx/ucx/blob/461292a7ecc6416910eb8ca54c0096b8b8d579b5/src/ucs/sys/topo.h#L64